x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -2712,8 +2737,8 @@ describe('Task Runner', () => {
 
     const runnerResult = await taskRunner.run();
 
-    // 1x(.server-log) and 2x(any-action) per alert
-    expect(actionsClient.enqueueExecution).toHaveBeenCalledTimes(5);
+    // 1x(.server-log) and 1x(any-action) per alert
+    expect(actionsClient.bulkEnqueueExecution).toHaveBeenCalledTimes(2);
```

Comment:
I would expect `bulkEnqueueExecution` to have been only called once since we have less than 1000 actions. It doesn't seem like we're calling it per action type right? Am I looking at this test right?

Reply:
I think it is because the actions are scheduled twice on lines `2688` and `2687`.